### PR TITLE
add abook config for prod

### PR DESCRIPTION
**Pull request by raymondng**
@stkem @andrewconner 
Added abook.conf && removed constraints

### CODE REVIEW

File: kifi-backend/modules/common/conf/prod/abook.conf [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# ~~~~~~~~~~~~~~~~~~
+# Urban Airship
+# ~~~~~~~~~~~~~~~~~~
+urban-airship {
+  key = "Bv482qzkSTWz4UNPf-IGoQ"
+  secret = "iiF74VvcTRy5mwPjCatESg"
+}
+
+airbrake {
+  key = "b903a091b834929686b95673c23fcb0d"
```
Comment (eishay) [inline kifi-backend/modules/common/conf/prod/abook.conf]: its the same airbrake key of eliza. please create a new airbrake project and use a new key.

File: kifi-backend/modules/common/conf/prod/abook.conf [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+# abook database
+  driver=com.mysql.jdbc.Driver
+  url="jdbc:mysql://abook.c1jga8odi83l.us-west-1.rds.amazonaws.com:3306/abook?autoReconnect=true&useUnicode=yes&characterEncoding=UTF-8"
+  user=abook
+}
+
+# ~~~~~~~~~~~~~~~~~~
+# Urban Airship
+# ~~~~~~~~~~~~~~~~~~
+urban-airship {
```
Comment (eishay) [inline kifi-backend/modules/common/conf/prod/abook.conf]: do you need the airship?

File: kifi-backend/modules/common/conf/prod/abook.conf (hunk 1 above)
Comment (raymondng) [inline kifi-backend/modules/common/conf/prod/abook.conf]: Updated. Thx.

File: kifi-backend/modules/common/conf/prod/abook.conf (hunk 2 above)
Comment (raymondng) [inline kifi-backend/modules/common/conf/prod/abook.conf]: Removed it. Thanks.

**Pull request merged**